src/App.js

```diff
@@ -18,7 +18,8 @@ function App() {
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/reviews" element={<Reviews />} />
-
+        <Route path="/categories" element={<Categories />} />
+        <Route path="/reviews/:category" element={<Reviews />} />
```

Comment:
This may cause issues when you go to do reviews by ID. Be careful how you structure your paths. You can only have one path on /reviews/ for example /reviews/1 will not work if you also have a path of /reviews/role-playing

Reply:
I've edited this so it's now /reviews/category/:category, is this good to merge now?